demo_data/README.md

```diff
@@ -31,11 +36,6 @@ demo_data
 └── simulated_no_bias.bam.bai
 ```
 
-Make sure files in demo_data/ have global read permission:
```

Comment:
Shall we add instructions about creating the output file

```
mkdir -v demo_output
```

Reply:
Should we just create the directory instead?

Reply:
Yep we could do that. Let’s see if an empty folder can be tracked by git or if we’ll need a clean placeholder.

Reply:
> Let’s see if an empty folder can be tracked by git.

It cant but I can edit the Nextflow pipeline that if it cant find the "output_dir" it creates it itself.

Reply:
We’d prefer users to create the `output_dir` themselves to avoid issues like typos or other unexpected errors.

Alternatively we could introduce an `output_dir_create` boolean option (default: False) so users can enable this if they want the workflow to create the output folder automatically.

Reply:
> Alternatively we could introduce an output_dir_create boolean option (default: False)

I dont prefer that. OK lets keep the current functionality but ask users to create output directory before running demo then.